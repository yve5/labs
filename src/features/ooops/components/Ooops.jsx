import React from 'react';
import { Link } from 'react-router-dom';
import { HOME_URL } from '../../home/resources/constants';

const Ooops = () => (
  <div className="container">
    <Link className="container__nav" to={HOME_URL}>
      Back to home
    </Link>

    <h1>Ooops!!!</h1>

    <p>This page does not exist...</p>
  </div>
);

export default Ooops;
