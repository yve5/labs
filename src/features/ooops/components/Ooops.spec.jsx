import { create } from 'react-test-renderer';
import { HashRouter } from 'react-router-dom';
import Ooops from './Ooops';

describe('Ooops', () => {
  it('should match expected snapshot. default', () => {
    const component = create(
      <HashRouter>
        <Ooops />
      </HashRouter>
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
