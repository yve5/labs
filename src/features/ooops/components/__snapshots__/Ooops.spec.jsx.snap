// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ooops should match expected snapshot. default 1`] = `
<div
  className="container"
>
  <a
    className="container__nav"
    href="#/"
    onClick={[Function]}
  >
    Back to home
  </a>
  <h1>
    Ooops!!!
  </h1>
  <p>
    This page does not exist...
  </p>
</div>
`;
